### ISSUE TITLE
Replace if else statement in utils.py with try except block 

### ISSUE THREAD

**Issue by vddesai1998** [HTTP-APIs/hydrus#184]
### I'm submitting a 
- [x ] bug report.
- [ ] feature request.

### Current Behaviour:

On each request we are making some calls to functions in utils.py to get objects (like api_doc).
In utils.py we are having an unnecessary if-else block in every get_* methods which checks the if condition whenever we try to get an object
like this
```
def get_doc():
    """Get the server API Documentation."""
    apidoc = getattr(g, 'doc', None)
    if apidoc is None:
        apidoc = doc_writer_sample.api_doc
        g.doc = apidoc
    return apidoc
```

### Expected Behaviour:

We can replace it with try except block and it would not give exception most of the times because we are setting these variables at the start of the server and we do not have to check those conditions on every request.
Like this
```
def get_doc():
    try:
        apidoc =  getattr(g, 'doc', None)
    except KeyError:
        apidoc = g.doc = doc_writer_sample.api_doc
    return apidoc
```
### Do you want to work on this issue?

Yes

**Comment by ashwani99** [comment]
> apidoc =  getattr(g, 'doc', None)
This would not work as if the `doc` attribute is not found then `apidoc` will be assigned value `None`. Also it will not throw `KeyError`. A solution for this can using `try-except` block would be:
```python
def get_doc():
    try:
        apidoc = getattr(g, 'doc')
    except AttributeError:
        g.doc = apidoc = doc_writer_sample.api_doc
    return apidoc
```

**Comment by xadahiya** [comment]
Fixed with #186
